bitnami/spark/values-production.yaml

```diff
@@ -124,6 +132,11 @@ master:
 ## Spark worker specific configuration
 ##
 worker:
+  #podAnnotations:
+  #  prometheus.io/path: "/metrics/prometheus"
+  #  prometheus.io/scrape: "true"
+  #  prometheus.io/port: "8081"
+
```

Comment:
Same as in the `master`:

Just this is fine, as we do in other containers. Remember that pod annotations are not exclusive for metrics use (we are adding `metrics.podAnnotations` for that):

```  
## Additional pod annotations
## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
##
podAnnotations: {}
```

And move it to the end of the `worker`